components/form/SelectOrCreateAuthSecret.vue

```diff
@@ -256,11 +272,14 @@ export default {
             'ssh-privatekey': base64Encode(this.privateKey),
           };
         } else {
-          secret._type = SECRET_TYPES.BASIC;
+          secret._type = this.allowCacerts ? SECRET_TYPES.OPAQUE : SECRET_TYPES.BASIC;
           secret.data = {
             username:  base64Encode(this.username),
             password: base64Encode(this.password),
           };
+          if (this.allowCacerts && this.cacerts) {
+            secret.data.cacerts = base64Encode(this.cacerts);
```

Comment:
I'm confused why there's a cacert getting tied to the credential secret?  There's already a separate place for that in the gitrepo.  @StrongMonkey ?

Reply:
Yeah, I will add that in 2.6. It should be able to reuse that from existing CABundle field.

Reply:
Ok so @mantis-toboggan-md you can drop basically all these changes in this file and just have a 2nd select-or-create that's like the first.

Reply:
Alright, done